evm_loader/program/src/transaction.rs

```diff
@@ -140,4 +140,15 @@ pub fn find_sysvar_info<'a>(accounts: &'a [AccountInfo<'a>]) -> Result<&'a Accou
 
     debug_print!("sysvar account not found");
     Err(ProgramError::InvalidInstructionData)
+}
+
+pub fn find_rent_info<'a>(accounts: &'a [AccountInfo<'a>]) -> Result<&'a AccountInfo<'a>, ProgramError> {
```

Comment:
This function does not belong to transactions module. It should be in utils or in entrypoint itself.